core/src/main/kotlin/org/evomaster/core/problem/rest/service/AbstractRestSampler.kt

```diff
@@ -253,6 +253,11 @@ abstract class AbstractRestSampler : HttpWsSampler<RestIndividual>() {
         if (swagger.paths == null) {
             throw SutProblemException("There is no endpoint definition in the retrieved Swagger file")
         }
+        // Onur: to give the error message for invalid swagger
+        else if (swagger.paths.size == 0){
+            throw SutProblemException("The swagger file ${configuration.bbSwaggerUrl.toString()} " +
```

Comment:
no need for the `.toString()`, as it is already a string... didn't IntelliJ flag it out?